src/spryker/docker/image/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,5 @@
+
+# https://github.com/elastic/elasticsearch-docker
+FROM docker.elastic.co/elasticsearch/elasticsearch:5.6.14
```

Comment:
Elasticsearch 5.6 is EOL as of 2019-03-11 according to https://www.elastic.co/support/eol - does a newer version work with Spryker?

It looks like the ES images have migrated back to https://hub.docker.com/_/elasticsearch thankfully, so could use that source (and the `5.6` tag instead of an older version of `5.6` release to pick up fixes)